tastypie_swagger/urls.py

```diff
@@ -1,13 +1,29 @@
+from django import __version__ as django_version
```

Comment:
Probably cleaner to call [`get_version`](https://docs.djangoproject.com/en/1.9/intro/install/#verifying)